Assets/Plugins/Android/EOSAndroidConfig.cs

```diff
@@ -34,6 +34,7 @@ public class EOSAndroidConfig : ICloneableGeneric<EOSAndroidConfig>, IEmpty
 
         public EOSConfig overrideValues;
 
+        public bool linkLibStatically = false;
```

Comment:
As mentioned in the PlatformSpecificConfigEditorAndroid comment, this should preferably be in it's own IEOSPluginEditorConfigurationSection for android options